qucs/qucs/components/wprobe.cpp

```diff
@@ -0,0 +1,84 @@
+//Created by Pedro Macedo
```

Comment:
Please use the proper recommended license header in all sources you created:
https://github.com/Qucs/qucs/wiki/Development#license-header